### DISCUSSION TITLE
Potential bug using Largo tool and is Biigle downtime scheduled today? (internal server error message)

### DISCUSSION THREAD

**Discussion by emmeb2021** [biigle/core#774] [Q&A] [answered]
Hi there!
Is the Biigle downtime scheduled ? ( internal error message when accessing Biigle.de at 01:27 GMT 09FEB 2024)

Regarding the issue with Largo, at the volume level using the Largo tool in my project named BENCHMARK project #1244

I was re-annotating the label POR_TU_012, I  selected all the annotations with this label to re-label them. I changed my mind on a few labels, assigning them on the next page back to the POR_TU_012 label. These annotations were not retained and there was no " are you sure you want to delete X labels" error box to let me know they would be dropped. I was able to re-annotate them okay so no need to help me on that but I did not recall this ever being an issue for me doing this action before?

Thanks!
Emme

**Comment by mzur** [comment, marked as answer]
Hi Emme, sorry for the late reply but I was ill. I could only patch BIIGLE back together (no, the downtime was not planned) but support requests had to wait.

Thanks a lot for reporting the Largo bug! This was indeed the worst kind of bug where annotations could be deleted without warning. I pushed a fix (https://github.com/biigle/largo/pull/149) that will be deployed in a few minutes.

**Reply by emmeb2021** [reply]
Glad your feeling better! Thanks for all the great work you guys do with Biigle, makes our job considerably easier (and fun!!) 
Em
